### PR TITLE
fix(migrations) add an initContainer for migrations up

### PR DESCRIPTION
**Pull request by hutchic**

### CODE REVIEW

File: kong-control-plane-cassandra.yaml [hunk 1]
```diff
@@ -75,6 +75,15 @@ spec:
...
         command: [ "kong", "migrations", "bootstrap" ]
+      - name: kong-migration-up
+        image: kong
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: KONG_PG_PASSWORD
+            value: kong
+          - name: KONG_PG_HOST
+            value: postgres.kong.svc
+        command: [ "kong", "migrations", "up" ]
```
Comment (hbagdi) [inline kong-control-plane-cassandra.yaml]: This is not sufficient. You need to do `kong migrations finish` as well.
Also, we should do this using a `Job` and not as an init-container.

Comment (hutchic) [reply kong-control-plane-cassandra.yaml]: does doing this as a `Job` apply to bootstrap as well?

Comment (hbagdi) [reply kong-control-plane-cassandra.yaml]: Yes. Please have a look at our Helm chart on how we are doing it:
https://github.com/helm/charts/tree/master/stable/kong/templates

We do a pre-migration and a post-migration for this purpose.

Comment (hutchic) [reply kong-control-plane-cassandra.yaml]: done in https://github.com/Kong/kong-dist-kubernetes/pull/97

**Pull request merged**